platform/mellanox/mlnx-fw-upgrade.j2

```diff
@@ -69,6 +70,7 @@ function ParseArguments() {
             ;;
             -v|--verbose)
                 VERBOSE_LEVEL="${VERBOSE_MAX}"
+                MFT_DIAGNOSIS_FLAGS="FLASH_ACCESS_DEBUG=1 FW_COMPS_DEBUG=1"
```

Comment:
@stephenxs imo, this is not a verbose mode but rather a debug mode. Suggest having a separate key.

Reply:
hi @nazariig  
thanks for the comments. we discussed it with the team and decided to keep it as it is because we need to log all messages when MFT tool fails and it doesn't bring benefit to introduce a new option.

Reply:
@nazariig 
yes. at least, it prints more with the options. but according to MFT team, it "won't be too long".
can you remind me what the concern is?